packages/realm/RealmJS.podspec

```diff
@@ -5,6 +5,8 @@ package = JSON.parse(File.read(File.expand_path('package.json', __dir__)))
 
 app_path = File.expand_path('../..', __dir__)
 
+build_realm_core = ENV["BUILD_REALM_CORE"] == "1"
```

Comment:
Perhaps the env could be `REALM_BUILD_CORE` to be more consistent with the other envs by using `REALM` as a prefix?